src/main/java/de/dennisguse/opentracks/io/file/exporter/TrackExportThread.java

```diff
@@ -0,0 +1,31 @@
+package de.dennisguse.opentracks.io.file.exporter;
+
+import android.content.Context;
+
+import java.io.OutputStream;
+
+import de.dennisguse.opentracks.content.data.Track;
+import de.dennisguse.opentracks.content.provider.ContentProviderUtils;
+import de.dennisguse.opentracks.io.file.TrackFileFormat;
+import de.dennisguse.opentracks.io.file.exporter.TrackExporter;
+
+public class TrackExportThread extends Thread {
```

Comment:
Can be removed (see other comment)

Reply:
it does not work the way you suggested (see other comment)